cfme/services/catalogs/catalog_item.py

```diff
@@ -145,17 +131,18 @@ def create(self):
         sel.wait_for_element(basic_info_form.name_text)
         catalog = fakeobject_or_object(self.catalog, "name", "Unassigned")
         dialog = fakeobject_or_object(self.dialog, "name", "No Dialog")
-        sel_provider = None
-        if self.item_type is not "Generic":
-            if self.provider.type_tclass is not "infra":
-                sel_provider = self.provider
+        # Provider is only in the create form for AnsibleTower type
+        select_provider = None
+        if not isinstance(self.provider, NoneType) \
+                and 'AnsbileTower' in [self.item_type, self.provider_type]:
```

Comment:
@mshriver : The provider field needs to be selected for orchestration provisioning (ec2 and heat ) and ansible , that means cloud and ansible , so filtering out infra and generic types.